frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -377,27 +381,31 @@ export const GRAPH_DISPLAY_VALUES_SETTINGS = {
     getHidden: (series, vizSettings) =>
       vizSettings["stackable.stack_type"] === "normalized",
     default: false,
+    inline: true,
+    marginBottom: "1rem",
   },
   "graph.label_value_frequency": {
     section: t`Display`,
     title: t`Values to show`,
-    widget: "radio",
+    widget: "segmentedControl",
     getHidden: (series, vizSettings) =>
       vizSettings["graph.show_values"] !== true ||
       vizSettings["stackable.stack_type"] === "normalized",
     props: {
       options: [
-        { name: t`As many as can fit nicely`, value: "fit" },
+        { name: t`Some`, value: "fit" },
         { name: t`All`, value: "all" },
       ],
+      variant: "fill-background",
+      inactiveColor: "text-dark",
     },
     default: "fit",
     readDependencies: ["graph.show_values"],
   },
   "graph.label_value_formatting": {
     section: t`Display`,
-    title: t`Value formatting`,
-    widget: "radio",
+    title: t`Auto formatting`,
```

Comment:
I made a comment regarding the copy change [on Notion](https://www.notion.so/Improve-the-visualization-settings-user-experience-8e1fb8fadb3f4cd9ba79e3bad7b2847b?d=6e7d23468e524f399ed6fe26a3b89eac#eaf83999d9ca4312afbc444cf56b659c).